### PR TITLE
fix: search shortcuts and formatting

### LINKED ISSUE #1: Fix search shortcut not working
`/` shortcut to access the search bar is not working.

KeyCodes are compared instead of just key. This means that swiss keyboards are unable to focus the search bar.

### PR DESCRIPTION
**Pull request by spaenleh**
This PR fixes the shortcut keys to the search functionality.

Modifications:
- `search.js`: 
  - format file using prettier
  - replace deprecated `keyCode` ([see docs](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)) by `key` which has better support for different keyboard layouts and locales

closes #1

### CODE REVIEW

File: sass/common/_variables.scss [hunk 1]
```diff
@@ -16,10 +16,10 @@ $yellow:   #ffe000;
 $black:    #1d2d35;
 $beige:    #fbf7f0;
 // $red:      #e55235;
-$purple:   #5d2f86;
+$purple:   #9444db;
```
Comment (marindeaudouce) [inline sass/common/_variables.scss:19]: Add a screenshot comparison of these new colors :)

**Review by marindeaudouce — CHANGES_REQUESTED** [review]
I would prefer having screenshots of the colors you modified :)
However it's really good! Even if 3/4 of the things you modified were indentation ;)

**Comment by spaenleh** [comment]
Thank you for the review @marindeaudouce 
Here are the requested comparison between the colors:

Old dark:
Capture d’écran 2022-10-15 à 15 07 04
New dark:
Capture d’écran 2022-10-15 à 15 06 56

Old light:
Capture d’écran 2022-10-15 à 15 06 42
New light:
Capture d’écran 2022-10-15 à 15 06 49

**Review by marindeaudouce — APPROVED** [review]
thank you (with no caps)

**Pull request merged**